bokehjs/src/coffee/models/layouts/layout_dom.coffee

```diff
@@ -207,26 +207,26 @@ class LayoutDOM extends Model
       'whitespace-left' : @_whitespace_left
       'whitespace-right' : @_whitespace_right
     }
-    if @responsive is 'stretch_both'
+    if @sizing_mode is 'stretch_both'
       constrained_variables = _.extend(constrained_variables, {
         'width': @_width
         'height': @_height
       })
-    if @responsive is 'scale_width'
+    if @sizing_mode is 'scale_width'
       constrained_variables = _.extend(constrained_variables, {
         'width': @_width
       })
-    if @responsive is 'scale_height'
+    if @sizing_mode is 'scale_height'
       constrained_variables = _.extend(constrained_variables, {
         'height': @_height
       })
     return constrained_variables
 
   @define {
-      height:   [ p.Number, null ]
-      width:    [ p.Number, null ]
-      disabled: [ p.Bool, false ]
-      responsive: [ p.Responsive, null ]
+      height:      [ p.Number              ]
+      width:       [ p.Number              ]
+      disabled:    [ p.Bool,       false   ]
+      sizing_mode: [ p.SizingMode, "fixed" ]
```

Comment:
hmmmm.....at some point during layout I decided to not set a default responsive/sizing_mode on the LayoutDOM but to Override it lower down the chain. This is why there's an override on Widget.

Was the change to move it up to to LayoutDOM deliberate?

If I recall correctly, which I may not be, the reason was because setting the default to LayoutDOM isn't really correct. There are a number of our LayoutDOM items that want to be in box/stretch_both mode for good reasons. In particular PlotCanvas and Toolbar spring to mind.